### PR TITLE
D8CORE-334: Responsive Image Styles

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Updates breakpoint settings
- Updates responsive images
- Adds new responsive images for Card and CTA

# Review By (Date)
- Oct 18

# Urgency
- Low

# Steps to Test

1. Review Code
2. Check out this branch
3. Do a full Drupal site install
4. Run tests
5. Do a manual review of image styles

# Affected Projects or Products
- Hero (banner) paragraph
- More to come as I swap out the image style in the paragraph later

# Associated Issues and/or People
- D8CORE-334
-https://github.com/SU-SWS/acsf-cardinalsites/pull/29
- https://github.com/SU-SWS/stanford_profile/pull/25

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/install/image.style.breakpoint_lg_1x.yml [hunk 1]
```diff
@@ -9,6 +11,6 @@ effects:
     id: image_scale
     weight: 1
     data:
-      width: 1024
+      width: 992
       height: null
-      upscale: false
+      upscale: true
```
Comment (pookmish) [inline config/install/image.style.breakpoint_lg_1x.yml]: Upscaling? :grimacing: That has never been a good thing IMO

Comment (sherakama) [reply config/install/image.style.breakpoint_lg_1x.yml]: Desired? No. Problem solving? Somewhat.

This is open for discussion as it does create grainy images but with the especially large 2X 2XL breakpoint (3000px wide) this solves a problem of not having enough pixels when a user inputs a small image. The upscaling will likely have a mitigated effect as image will only display to the max size of the screen and the browser does a good job of downsizing on the smaller screen sizes. Do we need to pull in the other devs and have a quick mind meld on this?

Comment (pookmish) [reply config/install/image.style.breakpoint_lg_1x.yml]: yeah lets have a working huddle. Maybe throw some different images at it too.

**Comment by pookmish** [comment]
These changes would need to get wrapped into [stanford_profile](https://github.com/SU-SWS/stanford_profile/) for a site install to display them

**Comment by sherakama** [comment]
Ok, in short, the resolution is to use upscaling for rigid or fixed-width styles and to not use upscaling for default, fluid, or flexible width options.

**Comment by sherakama** [comment]
@pookmish revised styles.

**Pull request merged**